sharding-transaction/sharding-transaction-base/sharding-transaction-base-seata-at/src/main/java/org/apache/shardingsphere/transaction/base/seata/at/Constant.java

```diff
@@ -17,31 +17,10 @@
 
 package org.apache.shardingsphere.transaction.base.seata.at;
 
-import io.seata.core.context.RootContext;
-import org.apache.shardingsphere.underlying.executor.engine.ExecutorDataMap;
-
-import java.util.Map;
-
 /**
- * Seata transaction broadcaster.
+ * Constant.
  */
-class SeataTransactionBroadcaster {
-    
-    private static final String SEATA_TX_XID = "SEATA_TX_XID";
+final class Constant {
```

Comment:
Please do not use `Constant` as a class name, the scope of this name is too big, it is very easy conflict and misunderstanding with other modules

Reply:
ok, I have nine it to TransactionSQLExecuteHook